src/components/movie-page/movie-page.jsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const MoviePage = ({film}) => {
+  const {
+    title,
+    bigPoster,
+    moviePoster,
+    genre,
+    year,
+    description,
+    rating,
+    descriptionRating,
+    quantityVotes,
+    producer,
+    listActors
+  } = film;
```

Comment:
Декомпозицию можно делать на месте тоже
```suggestion
const MoviePage = ({
  film: {
    title,
    bigPoster,
    moviePoster,
    genre,
    year,
    description,
    rating,
    descriptionRating,
    quantityVotes,
    producer,
    listActors
  }
}) => {

```